modules/manage/pages/iceberg/access-iceberg-topics.adoc

```diff
@@ -0,0 +1,103 @@
+
+See xref:reference:properties/cluster-properties.adoc[Cluster Configuration Properties] for the full list of cluster properties to configure for a catalog integration.
+
+=== Example REST catalog configuration
+
+For example, if you have Redpanda cluster configuration properties set to connect to a REST catalog named `streaming`:
+
+[,yaml]
+----
+iceberg_catalog_type: rest 
+iceberg_rest_catalog_endpoint: http://catalog-service:8181 
+iceberg_rest_catalog_client_id: <rest-connection-user>
+iceberg_rest_catalog_client_secret: <rest-connection-password>
+----
+
+And you use Apache Spark as a processing engine, configured to use the `streaming` catalog:
+
+[,spark]
+----
+spark.sql.catalog.streaming = org.apache.iceberg.spark.SparkCatalog
+spark.sql.catalog.streaming.type = rest
+spark.sql.catalog.streaming.uri = http://catalog-service:8181
+----
+
+Using Spark SQL, you can query the Iceberg table directly by specifying the catalog name:
+
+[,sql]
+----
+SELECT * FROM streaming.redpanda.<topic-name>;
+----
+
+Spark can use the REST catalog to automatically discover the topic's Iceberg table. 
+
+Depending on your processing engine, to xref:manage:iceberg/query-iceberg-topics.adoc[query the Iceberg data] you may need to also create a new table in your data warehouse or lakehouse. For an example, see xref:manage:iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc[].
+
+== Integrate file-system based catalog (`object_storage`)
+
+By default, Iceberg topics use the file-system based catalog (config_ref:iceberg_catalog_type,true,properties/cluster-properties[`iceberg_catalog_type`] cluster configuration set to `object_storage`). Redpanda stores the table metadata in https://iceberg.apache.org/javadoc/1.5.0/org/apache/iceberg/hadoop/HadoopCatalog.html[HadoopCatalog^] format in the same object storage bucket or container as the data files.
+
+If using the `object_storage` catalog type, you provide the object storage URL to an Iceberg client so it can access the catalog and data files for your Redpanda Iceberg tables.
+
+=== Specify metadata location
+
+The config_ref:iceberg_catalog_base_location,true,properties/cluster-properties[`iceberg_catalog_base_location`] property stores the base path for the file-system based catalog if using the `object_storage` catalog type. The default value is `redpanda-iceberg-catalog`. 
+
+CAUTION: Do not change the `iceberg_catalog_base_location` value after you have enabled Iceberg integration for a topic.
+
+If your engine needs the full JSON metadata path, use the following:
+
+```
+redpanda-iceberg-catalog/metadata/redpanda/<topic-name>/v<version-number>.metadata.json
+```
+
+This provides read access to a single snapshot (denoted by `version-number`) of a table. 
+
+NOTE: Redpanda automatically removes expired snapshots on a periodic basis. 
```

Comment:
nit: this maybe a better fit for the Query Iceberg Topics section

Reply:
Added link to a new time travel section in [Query Iceberg Topics](https://deploy-preview-989--redpanda-docs-preview.netlify.app/25.1/manage/iceberg/query-iceberg-topics/#query-using-time-travel) which also mentions snapshot expiry